corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -74,9 +74,14 @@ def filter_cases(self, xpath, additional_filters=None, auth=None, extra_instance
             "session_data": session_data,
         }
 
+        if use_formplayer:
+            url = settings.FORMPLAYER_URL + "/filter_cases"
+        else:
+            url = settings.XFORMS_PLAYER_URL
```

Comment:
@benrudolph curious if you think we should continue having the action contained in the content body or start routing the different requests to different URLs